test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -280,7 +286,14 @@
           [foo-service baz-service fail-service]
           (let [resp (http-client/get "http://localhost:8180/status/v1/simple")]
             (is (= 503 (:status resp)))
-            (is (= "error" (slurp (:body resp)))))))
+            (is (= "unknown" (slurp (:body resp)))))))
```

Comment:
I don't agree with this change. Seems like we're trying to soft-pedal things in a way that'll be confusing for users. What's the argument for reporting a status that potentially none of the services reported? Why do we want to collapse so many different status combinations into just returning "unknown"? Why would users rather see "unknown" and have to investigate rather than seeing the most severe status from among the reported statuses?